.travis.yml

```diff
@@ -1,8 +1,9 @@
+language: node_js
 os:
   - linux
   - windows
-language: node_js
 node_js:
+  - 'node'
```

Comment:
I get that, but then `11` wouldn't be included. I like it better when adding versions explicitly rather than re-adding previous versions when a new is released.